src/ManageCourses.Api/Services/Data/IDataService.cs

```diff
@@ -7,13 +7,16 @@ namespace GovUk.Education.ManageCourses.Api.Data
     public interface IDataService
     {
         void ProcessUcasPayload(UcasPayload payload);
-        OrganisationCourses GetCoursesForUserOrganisation(string email, string ucasCode); 
+        OrganisationCourses GetCoursesForUserOrganisation(string email, string ucasCode);
         IEnumerable<UserOrganisation> GetOrganisationsForUser(string email);
         UserOrganisation GetOrganisationForUser(string email, string instCode);
         void ProcessReferencePayload(ReferenceDataPayload payload);
         Course GetCourse(string email, string instCode, string ucasCode);
+        Course GetCourse(string instCode, string ucasCode);
```

Comment:
The 3 methods define here should not be here as 
1. DataService is getting needless big
2. DataService is just becoming at dumping ground.
3. DataService  is a dumping ground

Reply:
GetCourse => This is only serving a need for publishing
GetAllCourses => This is only serving a need for publishing

GetUcasInstitution => this is just a wrapper for context

LoadCourse => this looks like a static method